src/ocean/application/components/GCStats.d

```diff
@@ -39,6 +39,8 @@ public class GCStats
     }
     else
     {
+        pragma(msg, "There will be no gc stats! You need to use dmd transitional.");
```

Comment:
Could you add a link to where someone can find info about dmd-transitional?

Reply:
hmm... like this one? https://github.com/sociomantic-tsunami/dmd-transitional

Reply:
Yep.